### PR TITLE
Added spanish translation files for client and server side

### PR DESCRIPTION
**Pull request by criadoperez**

### CODE REVIEW

File: config/locales/server.es.yml
Comment (CvX) [inline config/locales/server.es.yml:1]:
```suggestion
es:
```

**Review by CvX — CHANGES_REQUESTED** [review]

**Comment by CvX** [comment]
@gschlager shall we merge this, pending changes? It pre-dates Crowdin integration. 🙂

**Comment by gschlager** [comment]
No, we can't merge it. Translations have to go through Crowdin. All changes would be overwritten by tomorrow's update.

@criadoperez Thanks for the PR, but we accept translation updates only through [Crowdin](https://translate.discourse.org/).

**Comment by criadoperez** [comment]
Ok, no problem. Thanks for the Crowdin link. I'll continue there.

**Comment by CvX** [comment]
Thank you! ☺️

**Pull request closed without merging**
